projects/1.20/assets/create-immersive-unnecessary-shaft/createimmersiveunnecessaryshaft/lang/zh_cn.json

```
@@ -0,0 +1,7 @@
+{
+  "block.createimmersiveunnecessaryshaft.treated_shaft": "防腐木传动杆",
+  "createimmersiveunnecessaryshaft.ponder.shaft.header": "使用防腐木传动杆在机械动力和沉浸工程间传递旋转力",
+  "createimmersiveunnecessaryshaft.ponder.shaft.text_1": "防腐木传动杆能直线传递旋转力",
+  "createimmersiveunnecessaryshaft.ponder.shaft.text_2": "它可以将沉浸工程和机械动力的机器相连，比如动能发电机",
+  "createimmersiveunnecessaryshaft.ponder.shaft.text_3": "相连的动能发电机会产生尽可能多的应力，但不会导致网络过载"

```

```suggestion
  "createimmersiveunnecessaryshaft.ponder.shaft.text_3": "相连的动能发电机会消耗尽可能多的应力，但不会导致网络过载"
```

---

坏，之前一直都以为应力这东西应该是网络的负载占用的量度，产生应力和消耗应力反过来理解了

跟随MCMOD中和游戏内部分文本的描述
```suggestion
  "createimmersiveunnecessaryshaft.ponder.shaft.text_3": "相连的动能发电机会产生尽可能多的应力影响值，但不会导致网络过载"
```